CSSTooltipPseudo/explainer.md

```diff
@@ -0,0 +1,897 @@
+the bounds of the browser window.
+* Styling built-in tooltips should produce consistent results across all browsers 
+on the same OS, ensuring an interoperable experience for authors.
+
+## Non-goals
+
+* This proposal is not meant to support tooltips with arbitrary HTML content, which
+can be covered by existing solutions, like `popover`, CSS Anchor positioning, and
+the not-yet-available `interesttarget` attribute.
+* Provide a means to set a [pointer](#tooltip-pointerarrow) on the `title` based
+tooltip. The solution should be specified to allow for this functionality in
+the future, though.
+
+## User research
+
+Use cases in this explainer were collected from the discussion in issue
+[8930](https://github.com/w3c/csswg-drafts/issues/8930). Additional inspiration
+was drawn from discussions in issues
+[9447](https://github.com/w3c/csswg-drafts/issues/9447), and OpenUI issue
+[730](https://github.com/openui/open-ui/issues/730).
+
+There are also many articles on the web that walk authors through various
+options for creating custom tooltips using tools available to them today.
+One such [article](https://blog.replaybird.com/css-tooltip-examples/)
+walks through different options, along with various examples and use cases
+for tooltips on the web.
+
+## Existing and upcoming tools available for custom tooltips
+
+If an author would like to adjust the style of their tooltip today, they must resort
+to one of the following alternative approaches to the built-in tooltip that they
+get with the `title` attribute.
+
+### Use an existing library or component
+
+Authors can use an existing library or component for a fully customizable tooltip.
+Some example libraries include:
+  - https://fluent2.microsoft.design/components/web/react/core/tooltip/usage
+  - https://www.opentip.org/
+  - https://floating-ui.com/?utm_source=popper.js.org
+  - https://www.npmjs.com/search?ranking=popularity&q=tooltip
+  - Many more   
+
+It is clear that there are many libraries and components for creating custom tooltips
+that authors can use to solve their need for more customization.
+
+If the library chosen is well architected to properly handle accessibility, this can
+also be a convenient way to get such functionality "for free". However, relying on
+external library to get accessibility right can be risky, which may limit the usability
+if not done correctly.
+
+The other downside of this approach is that it requires heavy use of scripting,
+which can be slower, and shouldn't be necessary if the author simply wants to
+adjust some basic styles.
+
+### Create a fully custom tooltip
+
+Authors can also choose to create their own custom tooltip or component, which
+requires their own input management, proper accessibility support and proper
+handling of the tooltip positioning, which can be complex.
+
+This does allow authors to acheive their goal of full cusomizability, but it often
+requires heavy use of scripting, or use of `:before` or `:after` pseudos, which
```

Comment:
```suggestion
requires heavy use of scripting, or use of `::before` or `::after` pseudos, which
```

(Pseudo-classes such as `:hover` start with a single colon. Pseudo-elements such as `::before` start with a double colon.)

Reply:
Whoops, thanks for the catch. Updated!